lib/dynamic_config/page_mode.rb

```diff
@@ -16,9 +12,10 @@ class PageModeBase
   # @return {string}
   def get(request)
     # If a session page mode is set the 'pm' cookie, return that,
-    # otherwise return the default page mode from DCDO.
-    (request && request.cookies[PAGE_MODE_KEY]) ||
-        DCDO.get('page_mode', DEFAULT_PAGE_MODE)
+    # otherwise return a sw or mc page mode
+    page_mode = rand(2) == 0 ? "feature-starwars" : "feature-mc"
+
+    (request && request.cookies[PAGE_MODE_KEY]) || page_mode
```

Comment:
what if the cookie has a bad value for some reason?  should we check against a whitelist, or like Caley suggests, just always have a fallback to one of the values?